pkgs/build-support/build-fhs-userenv/env.nix

```diff
@@ -124,6 +124,24 @@ let
     paths = [ etcPkg ] ++ basePkgs ++ targetPaths;
     extraOutputsToInstall = [ "out" "lib" "bin" ] ++ extraOutputsToInstall;
     ignoreCollisions = true;
+    postBuild = ''
+      if [ -d  $out/share/gsettings-schemas/ ]; then
+          # Create the standard schemas directory
+          rm -rf $out/share/glib-2.0
```

Comment:
Not sure how `buildFHSChrootEnv` works, will this not remove schemas from unfree deb packages symlinked to the environment?

Reply:
Hmm, I didn't think of that. The problem was that `share/glib-2.0` was getting linked as a n empty directory. Maybe I can check if the directory isn't a symlink.